### DISCUSSION TITLE
使用cdn引入全局UMD 版本的 naive  ui如何配置组件语言为中文？

### DISCUSSION THREAD

**Discussion by jingguanzhang** [tusen-ai/naive-ui#5998] [Q&A]
使用cdn引入全局UMD 版本的 naive  ui如何配置组件语言为中文？

**Comment by 79102576** [comment]
`
<n-config-provider :locale="zhCN" :date-locale="dateZhCN"></n-config-provider>
const { NConfigProvider, zhCN, dateZhCN } = naive;
const logAPP = createApp({
data() {
return {
zhCN:zhCN,
dateZhCN:dateZhCN
}}})`
